src/main/java/seedu/duke/logic/parser/AddCommandParser.java

```diff
@@ -79,6 +80,11 @@ private static Command parseAddLessonCommand(String userResponse) throws ParseEx
         if (flagMap.containsKey(LINK_FLAG)) {
             meetingUrl = flagMap.get(LINK_FLAG);
         }
+
+        if (LocalTime.parse(startTime).isAfter(LocalTime.parse(endTime))) {
+            throw new ParseException("Sorry, the starting time of a lesson needs to be before its ending time.");

```

Comment:
Perhaps you could add this message to the `Message` class and call it here as `throw new ParseException(Message.INVALID_XXX)` for consistency?